### DISCUSSION TITLE
simulation and design a solar fuel system with input CO2 and H2O ?

### DISCUSSION THREAD

**Discussion by peyman-mollaei** [DanWBR/dwsim#430] [Q&A]
Hello Professor dear Daniel Wagner

I am a graduate student of energy systems engineering and I am very interested in using the open-source software have developed to advance my thesis project. But I wanted to get your help.
The subject of my project is the production of liquid hydrocarbon fuel using solar thermal processes. In the system modeling section, I need to perform thermochemical redox reactions using solid metal oxides such as cerium oxide, and is it possible to simulate such a process using DWSIM? Thank you very much for your guidance

**Comment by DanWBR** [comment]
@peyman-mollaei I don't know your process enough but I guess that it is possible, yes.

**Reply by peyman-mollaei** [reply]
thank you, professor
For more details, I want to simulate thermochemical redox reactions that can also be used in the production of liquid hydrocarbon fuels. This process is known as the "hybrid thermochemical cycle" and involves combining two different chemical reactions, both of which are driven by heat.
In the first stage of the cycle, a metal oxide (known as a catalyst) is reduced using a reducing agent to produce metal with a low capacity of oxygen and oxygen gas. The metal oxide is then reacted with water and carbon dioxide in a second stage to produce hydrogen and carbon monoxide gases and the metal oxide is returns to the initial state for the next cycle.
The hydrogen and carbon monoxide produced in the second stage can then be combined in Fischer-Tropsch Synthesis to produce liquid hydrocarbon fuels such as methanol, gasoline, and diesel.

This process has several advantages, including producing liquid fuels using renewable energy sources, such as solar or geothermal power. Additionally, the process can be used to capture and recycle carbon dioxide, which can help reduce greenhouse gas emissions.
